shared/scripts/deploy-template.js

```diff
@@ -1,12 +1,13 @@
 const TemplatesDeployer = require('../lib/TemplatesDeployer')
+const { APPS } = require("../helpers/apps");
```

Comment:
Small suggestion, let's keep the single quotes and no semicolons :)
```suggestion
const { APPS } = require('../helpers/apps')
```